apps/happy-blocks/src/support-page/support-page-embed.tsx

```diff
@@ -0,0 +1,60 @@
+import { __, sprintf } from '@wordpress/i18n';
+import { SupportPageBlockAttributes } from './block';
+import { WordPressIcon } from './icon';
+import { InlineSkeleton } from './inline-skeleton';
+
+/**
+ * Rendered embed for the Support Page block.
+ */
+export const SupportPageEmbed = ( props: { attributes: SupportPageBlockAttributes } ) => {
+	const loaded = !! props.attributes.content;
+
+	const minToRead = sprintf(
+		/* translators: Minutes it takes to read embedded support page, eg: "5 min to read" */
+		__( '%1$d min read', 'happy-blocks' ),
+		props.attributes.minutesToRead ?? 0
+	);
+
+	const likes = sprintf(
+		/* translators: Number of people marked this page useful, eg: "25332 people have found this useful" */
+		__( '%1$d people have found this useful!', 'happy-blocks' ),
+		props.attributes.minutesToRead ?? 0
+	);
+
+	return (
+		<div className="hb-support-page-embed">
+			<div className="hb-support-page-embed__header">
+				<WordPressIcon variant="raster" />
+				<div>
+					<div className="hb-support-page-embed__title">
+						<InlineSkeleton loaded={ loaded }>
+							{ props.attributes.title }
+							{ /*<span className="hb-support-page-embed__badge">Support article</span>*/ }
```

Comment:
Do we need this commented out code?

Reply:
The original design has this badge. It is used to distinguish Support, Forums, and other content. However, so far we have only Support embeds, so David suggested hiding it. But the next step would be to add other content embeds, so this badge would be used.